packages/eslint-config/package.json

```diff
@@ -43,14 +42,13 @@
     "eslint-plugin-unicorn": "^51.0.1",
     "eslint-plugin-vitest": "^0.4.1",
     "eslint-plugin-you-dont-need-lodash-underscore": "^6.12.0",
-    "lodash.merge": "^4.6.2"
+    "lodash": "^4.17.21"
   },
   "peerDependencies": {
-    "eslint": "^8.0.0",
-    "prettier": "^3.0.0"
```

Comment:
i think we'll still want this since `@readme/eslint-config/prettier` is a thing, right?

Reply:
It is but it's just a config file and doesn't use Prettier.

Reply:
wait no actually but this is peer deps!

Reply:
if someone is installing this `eslint-config` package, they should probably install both eslint + prettier, right?